docs/checkstyle/check-docs.sh

```diff
@@ -7,6 +7,13 @@ cd docs/guides
 
 for docs in admin developer; do
   cd $docs
+  if test "$( grep -R opencast_major_version | grep '\w}\|{\w' )" != ""; then
```

Comment:
Not that important, but maybe replace `-R` with `-r` to not follow symlinks?

Also, this would fail for something like:

```
{{ opencast_major_version }} some text `{xy}`
```

Maybe check for `{opencast_major_version` instead?